contrib/components/registry/storage/pvc/registry-deploy-pvc-patch.yaml

```diff
@@ -0,0 +1,17 @@
+kind: Patch
+apiVersion: meta.k8s.io/v1
+target:
+  kind: DeploymentConfig
```

Comment:
Seems like you'd want patches to applied to a range of things.  "I want to patch a podspecy thing". "I want to patch metadata", "I want to patch these three kinds of objects".

Reply:
Those would be multiple patches.